remotestorage-module.js

```diff
@@ -1,4 +1,10 @@
-RemoteStorage.defineModule('myfavoritedrinks', function(privateClient) {
+// This variable will be reused in the other files
+// Enable change events for changes in the same browser window
```

Comment:
This shouldn't be instantiated in the module file.

Reply:
Thanks, I forgot to write a note about this! Since this project is using JS files without a build system, I would instantiate the remoteStorage variable in a separate file that's loaded first. Is that right?

Reply:
No, it should be done in the app code. And the module should be defined on `window.MyFavoriteDrinks` for example (that's what would happen with modules from npm as well).

Reply:
Ack, on it!